sdk/src/main/res/values-he/strings.xml

```diff
@@ -1,3 +0,0 @@
-<resources>
-	<string name="loading_text">מאמת שאתה אנושי</string>
```

Comment:
Why is this translation removed? Seems unrelated to the Issue.

Reply:
This was done in separate commit, which fixes link issues

Reply:
Why are there linting issues? Maybe due to the dependency updates? Seems unrelated to the Issue, please test the 3 updated languages that they still work.

Reply:
Testing still needs to be done, I will check after https://github.com/hCaptcha/HCaptcha-ios-sdk/issues/50

Reply:
@DSergiu this warning highlight actual problems. What changes was done:

1. `sdk/src/main/res/values-he/strings.xml` -> `sdk/src/main/res/values-iw/strings.xml` (hebrew
2. `sdk/src/main/res/values-id/strings.xml` -> `sdk/src/main/res/values-in/strings.xml` (indonesian)
3. `sdk/src/main/res/values-yi/strings.xml` -> `sdk/src/main/res/values-ji/strings.xml` (yiddish)

Before this change, Indonesian doesn't work at all (it shows English as fallback) I have tested actually device (Redmi Note 7) and emulator

Unfortunatelly I cannot test Yiddish, no such option on my device